src/font-size.scss

```diff
@@ -1,90 +1,90 @@
 @mixin font-size-5($line-height: regular) {
-  font-size: 5rem;
+  font-size: calc(5rem * var(--scale-factor, 1));
```

Comment:
Where is `--scale-factor` being set and does it have to be a css, not a sass variable?

Reply:
it can be set in the parent app like [this](https://github.com/RaspberryPiFoundation/projects-ui/blob/e26e5d036db14ef97b8ca8096918b4dcb5e41a91/src/index.scss#L24) but if it's not set it'll default to 1. I'll play with sass variables but I'm not sure what'll take precedence when it's redefined 🤔

Reply:
I can't seem to get the same default behaviour using sass variables. I either get the default 1.0 value used or a variable not defined error if I don't define it in that file 🤔 I've done a little searching too and annoying I can only find posts about theming with css rather than sass variables.
The [default value behaviour setup](https://sass-lang.com/documentation/variables/#default-values) _could_ work but it looks very setup for use within the same app and doesn't look like it would extend to web component well 😬